lib/syskit/roby_app/plugin.rb

```diff
@@ -146,8 +148,64 @@ def self.setup(app)
 
                 rtt_core_model = app.default_loader.task_model_from_name("RTT::TaskContext")
                 Syskit::TaskContext.define_from_orogen(rtt_core_model, register: true)
+
+                # Log Transfer FTP Server spawned during Application#setup
+                start_tmp_root_ca
+
+                start_local_log_transfer_server(
+                    app.log_dir,
+                    @tmp_root_ca.ca_user,
+                    @tmp_root_ca.ca_password,
+                    @tmp_root_ca.signed_cert_filepath
+                )
+            end
+
+            def self.send_file_transfer_command(name, tmp_root_ca, logfile)
+                # Checks if said process server exists and is connected
+                if process_servers[name]
+                    # Establishes communication with said process server
+                    client = Syskit::RobyApp::RemoteProcesses::Client.new(
+                            host: process_server[name].host_id,
+                            port: @server_port)
```

Comment:
To access process servers, use `Syskit.conf` (of type [Syskit::RobyApp::Configuration](https://www.rubydoc.info/github/rock-core/tools-syskit/master/Syskit/RobyApp/Configuration). In particular [process_server_config_for](https://www.rubydoc.info/github/rock-core/tools-syskit/master/Syskit/RobyApp/Configuration#process_server_config_for-instance_method) will check for existence and its `#client` attribute will return the client object, i.e.

~~~
client = Syskit.conf.process_server_config_for(name)
~~~